docs/how-to-build.md

```diff
@@ -4,244 +4,232 @@ slug: /how-to-build
-
 ## Prerequisites
 
-+ Linux or macOS operating system
-+ Git
-+ A Java Development Kit, version 8 to 17, installed in your environment to launch Gradle
-+ Python 3.8, 3.9, 3.10, or 3.11 to build the Gravitino Python client
-+ Optionally, Docker to run integration tests
-
-:::info Please read the following notes before trying to build Gravitino.
-
-+ Gravitino requires a minimum of JDK8 and supports up to JDK17 to run Gradle, so you need to install a JDK, version 8 to 17, to launch the build environment.
-+ Gravitino itself supports using JDK 8, 11, or 17 to build. The Gravitino Trino connector uses JDK17 to build (to avoid vendor-related issues on some platforms, Gravitino uses the specified Amazon Corretto OpenJDK 17 to build the Trino connector on macOS).
- You don't have to preinstall the specified JDK environment, as Gradle detects the JDK version needed and downloads it automatically.
-+ Gravitino uses the Gradle Java Toolchain to detect and manage JDK versions, and it checks the installed JDK by running the `./gradlew javaToolchains` command. See [Gradle Java Toolchain](https://docs.gradle.org/current/userguide/toolchains.html#sec:java_toolchain).
-+ Gravitino excludes all Docker-related tests by default. To run Docker-related tests, make sure you have installed Docker in your environment and either (1) set `skipDockerTests=false` in the `gradle.properties` file (or use `-PskipDockerTests=false` in the command) or (2) `export SKIP_DOCKER_TESTS=false` in the shell. Otherwise, all tests requiring Docker will be skipped.
-+ macOS uses `docker-connector` to make the Gravitino Trino connector work with Docker for macOS. See [docker-connector](https://github.com/wenjunxiao/mac-docker-connector), `$GRAVITINO_HOME/dev/docker/tools/mac-docker-connector.sh`, and `$GRAVITINO_HOME/dev/docker/tools/README.md` for more details.
-+ You can use OrbStack as a replacement for Docker for macOS. See [OrbStack](https://orbstack.dev/). With OrbStack, you can run Gravitino integration tests without needing to install `docker-connector`.
-+ Depending on how you deploy Gravitino, other software used in conjunction with Gravitino may contain known security vulnerabilities.
-:::
-
-## Quick start
+- Linux, macOS or Windows operating system
+- Git
+- JDK 8 (default), 11 or 17 for running Gradle, the build environment for Gravitino.
+- Python 3.8 (default), 3.9, 3.10, or 3.11, for building the Gravitino Python client
+- Docker (optional), for running the integration tests
 
-1. Clone the Gravitino project.
-
-   If you want to contribute to this open-source project, please fork the project on GitHub first. After forking, clone the forked project to your local environment, make your changes, and submit a pull request (PR).
+:::note
+Depending on how you deploy Gravitino, there may be contain known security vulnerabilities
+in other software used in conjunction with Gravitino.
```

Comment:
I missed this in my first review, and this shows why this is an issue. Breaking this sentence over two lines hides an error. Please keep the orignal text "Depending on how you deploy Gravitino, other software used in conjunction with Gravitino may contain known security vulnerabilities."

Reply:
Okay. If you hate breaking long lines, try figure out a way breaking this sentences to two.

Reply:
I don't think there is any need to do that either. BTW this was added as due to an ASF requirement.